textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcherTest.java

```diff
@@ -6,78 +6,163 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.IField;
 import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.common.Schema;
+import edu.uci.ics.textdb.common.constants.SchemaConstants;
 import edu.uci.ics.textdb.common.constants.TestConstants;
+import edu.uci.ics.textdb.common.field.DataTuple;
+import edu.uci.ics.textdb.common.field.ListField;
+import edu.uci.ics.textdb.common.field.Span;
 
 /**
  * @author zuozhi
- * @author laisycs
+ * @author shuying
  * @author chenli
  * 	
  * Unit test for RegexMatcher
  */
 public class RegexMatcherTest {
+	private Schema createSpanSchema(Schema schema) {
```

Comment:
There is another createSpanSchema() function inside RegexMatcher.java. Can you avoid this kind of duplicates?